src/commands/plugins/trust/verify.ts

```diff
@@ -50,6 +50,7 @@ export class Verify extends SfdxCommand {
       cacheDir: get(this.config, 'configDir') as string,
       configDir: get(this.config, 'cacheDir') as string,
```

Comment:
I know this isn't part of this PR but... why are `cacheDir` and `configDir` switched here?
Not sure how this affects things further in the code...
```suggestion
      cacheDir: get(this.config, 'cacheDir') as string,
      configDir: get(this.config, 'configDir') as string,
```